rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -627,6 +629,7 @@ extern "C" RocalImage ROCAL_API_CALL rocalVideoFileSource(RocalContext context,
                                                           RocalDecodeDevice rocal_decode_device,
                                                           unsigned internal_shard_count,
                                                           unsigned sequence_length,
+                                                          std::vector<std::string> file_names_list = {},
```

Comment:
pass as a reference instead. Applies to all others

Reply:
Done